topostats/entry_point.py

```diff
@@ -625,6 +625,13 @@ def create_parser() -> arg.ArgumentParser:
         required=False,
         help="Whether to ignore warnings.",
     )
+    process_parser.add_argument(
+        "--number-masks",
```

Comment:
This should be consistent with the parameter it modifies so either...

```suggestions
        "--number-masks",
```

...or if we got with `number_grain_plots` then...

```suggestions
        "--number-grain-plots",
        dest="number_grain_plots",
```